backend/functions/case-assignments/case.assignment.function.test.ts

```diff
@@ -5,25 +5,28 @@ import { AssignmentError } from '../lib/use-cases/assignment.exception';
 import { UnknownError } from '../lib/common-errors/unknown-error';
 import ContextCreator from '../lib/adapters/utils/application-context-creator';
 import { CaseAssignment } from '../../../common/src/cams/assignments';
-import { MockData } from '../../../common/src/cams/test-utilities/mock-data';
+import { MANHATTAN, MockData } from '../../../common/src/cams/test-utilities/mock-data';
 import { createMockAzureFunctionRequest } from '../azure/functions';
+import { CamsRole } from '../../../common/src/cams/session';
 
 describe('Case Assignment Function Tests', () => {
   const request = createMockAzureFunctionRequest({
     method: 'POST',
     query: {},
     body: {
-      caseId: '001-67-89123',
+      caseId: '081-67-89123',
       attorneyList: ['Bob Bob'],
       role: 'TrialAttorney',
     },
   });
   const context = require('azure-function-context-mock');
 
   beforeEach(async () => {
-    jest
-      .spyOn(ContextCreator, 'getApplicationContextSession')
-      .mockResolvedValue(MockData.getCamsSession());
+    jest.spyOn(ContextCreator, 'getApplicationContextSession').mockResolvedValue(
```

Comment:
Could we mock this the same way we do it in the case.assignment.controller.test beforeEach?

Reply:
Interesting thought.  Perhaps we should take a closer look at the controller test.